kevinbett/Flight-Booking-API
# #167941531 Deploys application to Heroku

## Resolves 167941531

## Description

  - This pull request hosts the application on Heroku

## How to test
- Run the application and test out the to endpoints using the remote url: `https://flight-api-bett.herokuapp.com/{desired end point}`

- The user should be able to execute all the endpoints remotely
